### DISCUSSION TITLE
Есть ли образец кода для вставки виджетов в тексте записи?

### DISCUSSION THREAD

**Discussion by nvda-ru** [maxsite/cms#462] [Q&A]
Здравствуйте, Максим:
Подскажите пожалуйста, где можно найти образец кода, при помощи которого можно вставлять виджеты в содержимое записи.
Например нужно вставить два одинаковых виджета категорий.
Может, есть что-то по аналогии вставки ушек при помощи php и шорткода.

Буду признателен, если покажете образцы.
В документации на сайте Maxsite не чего подобного не встретил.

**Comment by maxsite** [comment]
Вставляется как обычный php-код с помощью плагина **run_php**. Вот пример из компонента подвала шаблона MF:

```
<?php
if (function_exists('last_pages_unit_widget_custom'))
    echo last_pages_unit_widget_custom([
        'header' => '<h5 class="t-primary100 mar10-b">Последние записи</h5>',

        // значения prefs нужно указывать с новой строки!
        'prefs' => '
_order_asc = random
limit = 2
line1 = [thumb]
line2 = [title]
line3 =
line4 =
line5 =
page_start = <div class="mar15-tb flex">
page_end = </div>
title_start = <div class="flex-grow1 mar10-l">
title_end = </div>
block_start= <div>
block_end = </div>
content = 0
thumb_class = w100px-min reyes hover-no-filter
thumb_link_class =
thumb_width = 100
thumb_height = 80
exclude_page_allow = 0
exclude_page_add = 0
', '_footer']);
?>
```
